spec/lib/modules/aws_ip_ranges_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+
+RSpec.describe AWSIpRanges do
+  describe ".cloudfront_ips" do
+    before(:each) do
```

Comment:
I might be being anal, but I prefer us to just use `before do` since they do the same thing now

Reply:
Fair – although this is just reverting a commit to restore existing code!